### PR TITLE
docbook-dtds : bump to include docbook45

### PR DESCRIPTION
**Pull request by jimklimov**
Our asciidoc/a2x requires docbook45, among other things...

### CODE REVIEW

File: components/docbook/docbook/files/docbook-catalog-install.sh [hunk 1]
```diff
@@ -103,6 +103,31 @@ test ! -f /etc/sgml/xml-docbook.cat && \
     ( cd /etc/sgml; \
       ln -s xml-docbook-4.4-${Version}-${Release}.cat xml-docbook.cat )
```
Comment (pyhalov) [inline components/docbook/docbook/files/docbook-catalog-install.sh]: Remove unnecessary ln -s here and before this

File: components/nut/Makefile [hunk 2]
```diff
@@ -0,0 +1,67 @@
+#
```
Comment (pyhalov) [inline components/nut/Makefile]: nut doesn't belong to this PR

Comment (jimklimov) [inline components/nut/Makefile]: oops, thanks

2016-03-30 20:23 GMT+02:00 Alexander Pyhalov notifications@github.com:

**Comment by pyhalov** [comment]
BTW, I see the following after installing new packages:
ls -l  /etc/sgml/sgml-docbook.cat
lrwxrwxrwx 1 root root 29 Aug 24  2015 /etc/sgml/sgml-docbook.cat -> sgml-docbook-4.4-1.0-30.1.cat
ls -l  /etc/sgml/xml-docbook.cat  
lrwxrwxrwx 1 root root 28 Aug 24  2015 /etc/sgml/xml-docbook.cat -> xml-docbook-4.4-1.0-30.1.cat

Perhaps, there are issues in install script?

**Comment by jimklimov** [comment]
Thanks for the notice, should be fixed in latest commit.

2016-03-30 20:30 GMT+02:00 Alexander Pyhalov notifications@github.com:
